book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] According to the requirements, Getters and Setters for @rentals should be present. Kindly add the rentals to the attr_accessor instead of having it as attr_reader. You could modify your code to look like this: `attr_accessor :title, :author, :rentals` 👍🏽